src/networkremote/clientmanager.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CLIENTMANAGER_H
```

Comment:
Header guard should match the class name in uppercase

Reply:
Done